libraries/rush-lib/src/logic/RepoStateFile.ts

```diff
@@ -26,6 +27,10 @@ interface IRepoStateJson {
    * A hash of the CommonVersionsConfiguration.preferredVersions field
    */
   preferredVersionsHash?: string;
+  /**
+   * A hash of the injected dependencies in related package.json
+   */
+  packageJsonInjectedDependenciesHash?: string;
```

Comment:
🤔 I had thought that **repo-state.json** was only used when `preventManualShrinkwrapChanges` is enabled.  But apparently it is now always generated.

We should update the website docs to reflect this:

https://rushjs.io/pages/advanced/rush_files_and_folders/

![image](https://github.com/microsoft/rushstack/assets/4673363/36376ac6-81e5-4e73-9e3f-24a4edcee7c5)


I can make a PR